sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataView.java

```diff
@@ -75,7 +78,8 @@ protected void initView(String params) {
 			LayoutUtil.fluidColumnLoc(4, 0, 6, 0, TASKS_LOC),
 			LayoutUtil.fluidColumnLoc(4, 0, 6, 0, SAMPLES_LOC),
 			LayoutUtil.fluidColumnLoc(4, 0, 6, 0, EVENTS_LOC),
-			LayoutUtil.fluidColumnLoc(4, 0, 6, 0, SORMAS_TO_SORMAS_LOC));
+			LayoutUtil.fluidColumnLoc(4, 0, 6, 0, SORMAS_TO_SORMAS_LOC),
+			SurvnetGateway.layoutFragment());
```

Comment:
I would not hide the structure of the layout into a function.

It's better to see how the whole layout looks like then seeing some function names.

Reply:
I thought about only adding it only if needed; the layouting should remain in this class anyways.